src/util/util.ts

```diff
@@ -319,3 +320,23 @@ export function getSeverity(severity: number): vscode.DiagnosticSeverity {
       return vscode.DiagnosticSeverity.Warning;
   }
 }
+
+export function sonarCloudRegionToLabel(region: number): SonarCloudRegion {
```

Comment:
I decided to implement the conversion here instead of the language server because I would have to rewrite several classes to achieve it there